Metalama.Framework/Advising/IAdviceFactory.cs

```diff
@@ -166,7 +176,7 @@ public interface IAdviceFactory
         /// <param name="tags">An optional opaque object of anonymous type passed to the template method and exposed under the <see cref="meta.Tags"/> property of the
         ///     <see cref="meta"/> API.</param>
         /// <seealso href="@overriding-fields-or-properties"/>
-        IOverrideAdviceResult<IProperty> OverrideAccessors(
+        IOverrideAdviceResult<IPropertyOrIndexer> OverrideAccessors(
```

Comment:
We should split this into two methods (Petr proposed this) and have `IProperty OverrideAccessors(IFieldOrProperty)` and `IIndexer OverrideAccessors(IIndexer)`. This change breaks e.g. Metalama.Extensions.

Reply:
Ok I did it.